ci-operator/step-registry/etcd-encryption/conf/etcd-encryption-conf-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "etcd-encryption/conf/etcd-encryption-conf-ref.yaml",
```

Comment:
Go though etcd-encryption-conf-commands.sh, seem like no need `conf`subfolder, I suggest to remove the subfolder.

Reply:
Fixed